### PR TITLE
fix: handle EU server time issue

### PR DESCRIPTION
**Pull request by cdnninja**
Sometimes EU reports old time stamps

### CODE REVIEW

**Comment by cdnninja** [comment]
@ZuinigeRijder  you are thinking move this logic to the vehicle setting?

**Comment by ZuinigeRijder** [comment]
Yes, I can make a proposal (new) Pull request?

**Comment by cdnninja** [comment]
Of course! This is as much yours as mine! I don't have one of these cars anymore so can't say I spend much time on it.  I just increased your access to admin on this repo as well.

File: hyundai_kia_connect_api/ApiImplType1.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def _get_authenticated_headers(
     def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None:
         if get_child_value(state, "Date"):
             # `Date` field is in UTC time
-            if vehicle.last_updated_at:
+            if not vehicle.last_updated_at:
                 vehicle.last_updated_at = parse_datetime(
                     get_child_value(state, "Date"), dt.timezone.utc
```
Comment (ZuinigeRijder) [inline hyundai_kia_connect_api/ApiImplType1.py:54]: I think this was wrong, someone else already reported that this was wrong for ccs2, it should be self.data_timezone

**Review by ZuinigeRijder — CHANGES_REQUESTED** [review]

**Pull request closed without merging**